README.md

```diff
@@ -1,7 +1,7 @@
-Telephone is a VoIP program which allows you to make phone calls over
+The telephone is a VoIP program that allows you to make phone calls over
```

Comment:
"The telephone"? No, "Telephone" is a name here.

Reply:
hacktoberfest noise :(